docs/next/guides/cell-features/formatting-cells.md

```diff
@@ -94,8 +94,21 @@ To enable the custom borders feature, set the `customBorders` option. This can e
 
 To initialize Handsontable with predefined custom borders, provide the cell coordinates and border styles in form of an array:
 
-- with row/col pairs: `{ row: 2, col: 2, left: { /*...*/ } }`
-- or with range details: `{ range: { from: { row: 1, col: 1 }, to: { row: 3, col: 4 } }, left: { /*...*/ } }`
+- with row/col pairs: `{ row: 2, col: 2, start: { /*...*/ } }`
```

Comment:
Meta comment :) @budnix I think it would be easier for us to collaborate in the single PR https://github.com/handsontable/handsontable/pull/9141, would you agree?

Reply:
I agree, that's why the PR is based on the https://github.com/handsontable/handsontable/pull/9141 (`rtl-docs` branch). I think of the `rtl-docs` branch as epic of the docs changes for RTL. Anyway, I can merge the changes to #9141 and continue working on that branch.

Reply:
OK, let's do that! I still think that my comment above should be done https://github.com/handsontable/handsontable/pull/9199#discussion_r801798568, but it can be in #9141.

Reply:
I moved the meta-comment to the class description (API docs) https://github.com/handsontable/handsontable/pull/9199/commits/7d0f9a5d48f6c520add86303988fab79572fae4d.